plugins/rollbar_dashboards/app/views/rollbar_dashboards/_deploy.html.erb

```diff
@@ -1,9 +1,10 @@
-<% if deploy.project.rollbar_dashboards_settings.any? && deploy.succeeded? %>
+<% if (settings = deploy.project.rollbar_dashboards_settings.presence) && deploy.succeeded? %>
   <%# replaced by responsive_load.js.erb %>
   <%= content_tag(
       :span,
       '',
-      class: 'lazy-load',
+      class: 'lazy-load dashboard-container',
+      style: "min-height: #{rollbar_dashboard_placeholder_size(settings)}",
```

Comment:
I'd feel more comfortable if this was stating the unit explicitly like `em`

Reply:
Oops that should have been there. See '_project.html.erb'